odoo/openupgrade/doc/source/modules110-120.rst

```diff
@@ -638,6 +638,10 @@ missing in the new release are marked with |del|.
 |website_twitter                             |                                                 |
 +--------------------------------------------+-------------------------------------------------+
 
+.. [#account_asset] 'Account Asset' module it is replaced by the Odoo Community Association module
```

Comment:
s/it is/is (the subject is already there)

Reply:
yeah, it's a typo